lib/matplotlib/mathtext.py

```diff
@@ -2504,7 +2504,8 @@ def symbol(self, s, loc, toks):
                     break
             # Binary operators at start of string should not be spaced
             if (c in self._binary_operators and
-                    (len(s[:loc].split()) == 0 or prev_char == '{')):
+                    (len(s[:loc].split()) == 0 or prev_char == '{' or
```

Comment:
This should support everything in `_left_delim`.  Could probably just be `prev_char` in `self._left_delim`.